go.mod

```diff
@@ -1,3 +1,28 @@
-module github.com/kubernetes-sigs/kube-agentic-networking
+module sigs.k8s.io/kube-agentic-networking
 
-go 1.24.8
+go 1.24.2
+
+require (
+	k8s.io/apimachinery v0.34.2
+	sigs.k8s.io/controller-runtime v0.22.4
```

Comment:
Can we avoid relying on controller-runtime?

Reply:
Done.